account_invoice_validate_send_email/models/account_invoice.py

```diff
@@ -109,3 +115,12 @@ def action_send(self):
                 )
                 self.invoice_sent = True
         return True
+
+    @api.model
+    def _compute_yamato_slip_number(self):
```

Comment:
What about the point we talked about - we should comment how/why the 返品伝票 case is out of scope.

Reply:
Add if logic.
If invoice type is out_refund, except compute_slip_number step.
Now comment like below:
# This method not supposed red_order. Because red_order haven't delivery of own company.